jq.c

```diff
@@ -308,8 +315,18 @@ jdbc_jvm_init(const ForeignServer * server, const UserMapping * user)
 
 	if (FunctionCallCheck == false)
 	{
-		classpath = (char *) palloc0(strlen(strpkglibdir) + 19);
-		snprintf(classpath, strlen(strpkglibdir) + 19, "-Djava.class.path=%s", strpkglibdir);
+		const char* env_classpath = getenv("CLASSPATH");
+		int classpath_length = strlen(strpkglibdir) + 19;
+
+		if (env_classpath != NULL) {
+			classpath_length += 1 + strlen(env_classpath);
+			classpath = (char *) palloc0(classpath_length);
```

Comment:
Can you use `psprintf` instead of using `palloc0` and `snprintf`?<br>
Similar with line 326

Reply:
Done, tested locally and works for me :+1: